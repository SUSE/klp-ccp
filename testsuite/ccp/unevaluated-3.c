static int g(void)
{
	return 1;
}

static int h(void)
{
	return 1;
}

static int i(void)
{
	return 1;
}

static int j(void)
{
	return 1;
}

static int k(void)
{
	return 1;
}

static int l(void)
{
	return 1;
}

void pu_f(void)
{
	__builtin_choose_expr(0, i(), j());
	__builtin_choose_expr(1, k(), l());
}
