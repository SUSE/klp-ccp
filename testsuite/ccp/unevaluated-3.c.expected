/* klp-ccp: from testsuite/ccp/unevaluated-3.c */
static int i(void);

static int j(void)
{
	return 1;
}

static int k(void)
{
	return 1;
}

static int l(void);

void klpp_pu_f(void)
{
	__builtin_choose_expr(0, i(), j());
	__builtin_choose_expr(1, k(), l());
}
